rascal2/widgets/controls/model.py

```diff
@@ -0,0 +1,77 @@
+import warnings
+
+from pydantic_core import ValidationError
```

Comment:
If this is all we need from **pydantic_core**, might make sense to expose and grab it from RATapi somehow

Reply:
turns out you can import it from pydantic itself, which i need for model validation - will import it that way instead